cli/README.md

```diff
@@ -0,0 +1,135 @@
+## Cook CLI
+
+### Installation
+
+To install the Cook CLI, clone this repo and from this folder run:
+
+```bash
+python3 setup.py install
+```
+
+This will install the `cs` command on your system.
+
+### Configuration
+
+The Cook CLI client is designed with "end user ergonomics" in mind. 
+Custom defaults may be provided for all commands. 
+Multiple clusters are supported via configuration.
+
+In order to use the Cook CLI, you’ll need a configuration file. 
+`cs` looks first for a `.cs.json` file in the current directory, and then for a `.cs.json` file in your home directory. 
+The path to this file may also be provided manually via the command line with the `--config` option.
+
+The file looks something like this:
```

Comment:
Clarify that this project directory includes a `.cs.json` file. E.g.:
"There is a sample `.cs.json` file included in this directory, which looks something like this:"
(I was confused about whether there were defaults hard-coded in the tool, or if it was finding the defaults somewhere.)

Reply:
Done